website/content/docs/connect/ca/vault.mdx

```diff
@@ -210,30 +210,51 @@ The following Vault policy allows Consul to use pre-existing PKI paths in Vault.
 Consul is granted read-only access to the PKI mount points and the Root CA, but is
 granted full control of the Intermediate or Leaf CA for Connect clients.
 
-In this example the `RootPKIPath` is `connect_root` and the `IntermediatePKIPath`
-is `connect_inter`. These values should be updated for your environment.
+This example uses placeholder values for `RootPKIPath` and `IntermediatePKIPath`.
+Please update those placeholders to match the path values in the CA provider configuration.
 
 <CodeBlockConfig filename="vault-managed-pki-policy.hcl">
 
 ```hcl
-# Existing PKI Mounts
+# -----------------------------------------------------------------------------
+# Vault Managed PKI Mounts
+# -----------------------------------------------------------------------------
+
 path "/sys/mounts" {
   capabilities = [ "read" ]
 }
 
-path "/sys/mounts/connect_root" {
+# -----------------------------------------------------------------------------
+# Needed for Consul to renew its Vault token (if renewable) whether provided
+# directly in the CA provider config or using an auth method.
+
+path "auth/token/renew-self" {
```

Comment:
Question for reviewer: do we think we should move the auth token renewal stuff to the bottom of the list of permissions? Maybe the most important stuff is the CA stuff itself.

(I think I agree with this. Will probably move to bottom unless told otherwise.)